scripts/backup

```diff
@@ -1,27 +1,61 @@
 #!/bin/bash
 
-set -eu
 
-# Set app specific variables
-app="$YNH_APP_INSTANCE_NAME"
-dbname=$app
-dbuser=$app
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Source YunoHost helpers
+source _common.sh
```

Comment:
Please use this syntax
https://github.com/YunoHost/example_ynh/blob/master/scripts/backup#L9-L14
Here the script is not found.